main.go

```diff
@@ -8,17 +8,17 @@ import (
 
 // PageE creates a stealth page that can't be detected as bot.
 func PageE(b *rod.Browser) (*rod.Page, error) {
```

Comment:
Can you also update the name convention of the `PageE` and `Page`?

Reply:
Yea sure, will commit shortly